### DISCUSSION TITLE
夜间模式能否增加对首页是否开启的开关呢

### DISCUSSION THREAD

**Discussion by Robotxm** [the1812/Bilibili-Evolved#4025] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/5405331/221288758-304290db-229b-4257-a881-4e1739ba19cb.png)

目前在同时使用 Bilibili Evolved 脚本和 bilibili 搜索引擎首页样式这个 usercss。后者对于首页也提供了夜间模式，而二者的夜间模式在样式上会有冲突，能否加一个开关，使得可以单独针对首页关闭 Bilibili Evolved 的的夜间模式呢？

**Comment by Ethkuil** [comment]
相关：#3386

**Reply by Geeyun-JY3** [reply]
呃我当时开这个 issue 最主要的需求就是让 Bilibili Evolved 的`自定义顶栏`和 bilibili-simple-home 这个 UserCSS 的顶栏兼容下，因为以前兼容过，做法就是把 bilibili-simple-home 的这个顶栏给顶掉。

**Comment by Geeyun-JY3** [comment]
这就是 Bilibili Evolved 的`夜间模式`还没适配内测首页，不开这个 UserCSS 也是一样的：
![image](https://user-images.githubusercontent.com/49396421/221348740-8b11e9d3-8401-4114-8bdb-c7636ee56b30.png)

要么等适配，要么先退回新版首页。
